dll/shellext/fontext/lang/en-US.rc

```diff
@@ -13,6 +13,15 @@ BEGIN
     IDS_COL_NAME "Name"
 END
 
+STRINGTABLE
+BEGIN
+    IDS_COL_FILENAME "Filename"
+    IDS_COL_SIZE "Size"
+    IDS_COL_MODIFIED "Modified"
+    IDS_COL_ATTR "Attributes"
+    IDS_COL_ATTR_LETTERS "RHSAC"
```

Comment:
If these letters are standard and really don't change with the localization (as for example the arguments of the `attrib` command...), then you can make this string localizaton-independent and not copy it everywhere?

Reply:
I dont know if they are standard.